File: blog/src/types/friend.ts
export interface Friend {
  cover: string
  title: string
  introduction: string
  recommendStatus: boolean
  createTime: string
  url: string
}

export interface FriendGroup {
  [category: string]: Friend[]
}

export interface FriendDTO {
  cover: string
  title: string
  introduction: string
  url: string
}

File: yelingzi-blog/src/main/java/com/yeling/other/domian/entity/Friend.java
package com.yeling.other.domian.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Friend {

    private Integer id;

    private String title;

    private String cover;

    private String introduction;

    private Integer state;

    private LocalDateTime createTime;

    private String url;

    private Integer recommendStatus;

    private Integer userId;

    private String email;

}
